packages/browser-wallet/src/popup/pages/Account/ManageTokens/TokenList.tsx

```diff
@@ -113,8 +113,9 @@ const lookupTokenIdConfigure = (
 
             try {
                 const [token] = await getTokens(contractDetails, client, network, account, [searchQuery]);
-
-                value = [token as ContractTokenDetails];
+                if (token && token.metadata) {
+                    value = [token as ContractTokenDetails];
+                }
             } catch {
                 value = [];
             }
```

Comment:
Seems a bit weird to re-set the `value` to the value it already has? Maybe change to finally, and move the section below up in that block instead?

Reply:
I think it is okay to be explicit here. :shrug: